style/haskell/README.md

```diff
@@ -5,13 +5,16 @@ Haskell
 
 * Break long expressions before operators or keywords.
 * Break long type signatures between arguments.
+* Don't use a blank line between pragma and module statements.
```

Comment:
> How about dropping this guide suggestion for now; we can revisit it after we write more code?

I'm not terribly opposed to this, but I'd rather have a guideline and be consistent. Neither way is actually better. I feel like it will just be harder to pick one or the other later, as we'll have more code following different formats.

It looks like ghc-base (probably the largest repository of blessed Haskell code) uses a space after the pragma lines: https://github.com/ghc/packages-base

Any objection to using ghc as a reference for formatting when there's no obvious winner?